public/app/plugins/panel/pmm-update/components/AvailableUpdate/AvailableUpdate.tsx

```diff
@@ -25,10 +23,10 @@ export const AvailableUpdate: FC<AvailableUpdateProps> = ({ nextVersionDetails }
         {Messages.availableVersion}
         :&nbsp;
         <span data-testid="update-latest-version" className={styles.latestVersion}>
-          {showFullVersion ? nextFullVersion : nextVersion}
+          {showFullVersion ? nextVersion?.tag : nextVersion?.version}
         </span>
         <span data-testid="update-latest-release-date" className={styles.releaseDate}>
-          ({nextVersionDate})
+          ({nextVersion?.timestamp})
```

Comment:
Is ok to show the timestamp or a formatted date ?

Reply:
Good catch, fixed.